src/Hooks/getExerciseByID.ts

```diff
@@ -0,0 +1,17 @@
+import { useState } from 'react';
+import { Exercise } from '../Datatypes/datatypes';
+
+const URL_API = 'localhost:8000';
```

Comment:
[getAllExercises.ts](https://github.com/OnlineTrainingPlatform/UI/pull/21/files#diff-24c86d787d84c5d333b63f43eb34755723b038657118234d8550eabce8c92ba0) also has this binding. Maybe a unified class for handling API calls could be a good idea.